translate/translate.json

```diff
@@ -94387,6 +94402,7 @@
     },
     {
       "en": "batchupload",
+      "nL": "batch upload",
```

Comment:
If I remember this needs to stay the same as en because of a translate bug that affects p16filterevents, I'll check next week when I'm back :)

Reply:
accidentally used a capital letter